src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -330,8 +330,19 @@ using namespace Microsoft::Console::Interactivity;
                 // will return the console handle again, not the owning
                 // terminal's handle. It's not entirely clear why, but WS_POPUP
                 // is absolutely vital for this to work correctly.
-                const auto windowStyle = WS_OVERLAPPEDWINDOW | WS_POPUP;
-                const auto exStyles = WS_EX_TOOLWINDOW | WS_EX_TRANSPARENT | WS_EX_LAYERED | WS_EX_NOACTIVATE;
+                //
+                // We're doing WS_MINIMIZEBOX | WS_SYSMENU to get a "minimize
+                // button" for the window, but not the rest of WS_OVERLAPPED, so
+                // that we don't have an actual border, caption, nothing.

```

Comment:
Why do we need a minimize button? Does minimizing a window only work if there's a minimize button?

Reply:
YUP